sgx-tools/dcap_linux.go

```diff
@@ -0,0 +1,358 @@
+package main // import "github.com/inclavare-containers/sgx-tools"
+
+import (
+	"bytes"
+	"encoding/binary"
+	"encoding/hex"
+	"fmt"
+	"github.com/go-restruct/restruct"
+	"github.com/golang/protobuf/proto"
+	"github.com/inclavare-containers/rune/libenclave/intelsgx"
+	pb "github.com/inclavare-containers/sgx-tools/proto"
+	"github.com/sirupsen/logrus"
+	"net"
+)
+
+const (
+	aesmdSocket         = "/var/run/aesmd/aesm.socket"
+	attKeyIdLength      = 256
+	nonceLength         = 16
+	qeReportInfoLength  = 960
+	timeOut             = 15000000
+	defaultDcapAttKeyId = "0000000020008c4f5775d796503e96137f77c68a829a0056ac8ded70140b081b094490c57bff0000000000000000000000000000000001000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000020000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000"
+)
+
+func dialAesmd() (*net.UnixConn, error) {
+	addr, err := net.ResolveUnixAddr("unix", aesmdSocket)
+	if err != nil {
+		return nil, err
+	}
+
+	conn, err := net.DialUnix("unix", nil, addr)
+	if err != nil {
+		return nil, err
+	}
+
+	return conn, nil
+}
+
+func transmitAesmd(conn *net.UnixConn, req *pb.AesmServiceRequest) ([]byte, error) {
```

Comment:
For rfc commit, adding extra comments will be useful for reviewers, e.g, add a comment line to indicate this is a duplicated function inherited from intelsgx package.

Reply:
> For rfc commit, adding extra comments will be useful for reviewers, e.g, add a comment line to indicate this is a duplicated function inherited from intelsgx package.

Ok, Thanks.